test/plugins/features/movies/controller.test.js

```diff
@@ -2,18 +2,56 @@
 
 const Controller = require('../../../../lib/plugins/features/movies/controller');
 
-describe('movie controller', () => {
+describe('POST /movie controller', () => {
 
   describe('create', () => {
 
     it('creates a movie', async () => {
       const payload = { title: 'WALL-E' };
-
       const movie = await Controller.create(payload);
-
       expect(movie.get('title')).to.eql(payload.title);
+
     });
 
   });
 
 });
+
+describe('GET /movie controller', () => {
+
+  describe('find', () => {
+
+    it('returns an array of movies', async () => {
+      const query = {};
+      const movies = await Controller.find(query);
```

Comment:
nit: add a newline after this line for extra readability, since there's a separation in logic. typically for tests, the style I go for is:

```js
// define constant
// define another constant

// make function call that i'm testing
// make any other function call that i'm testing

// add expect call
// add another expect call
// add another expect call
```